src/app/components/copy-to-clipboard/index.html

```diff
@@ -0,0 +1,40 @@
+<sky-demo-page
+  pageTitle="Copy to clipboard"
+  skyuxModule="SkyClipboardModule"
+  packageName="lib-clipboard"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The copy to clipboard component creates a button on your page that enables users to copy the content to their computer's clipboard.
```

Comment:
I think we should delete "on your page." I mean, I assume this could be used in a modal, right?

Also, we should delete "the" before content since at this point in the description, it's just generic content, not a specific piece of content. And I'd also delete "computer's."